src/CVRF/cvrf_priv.c

```diff
@@ -1249,108 +1252,24 @@ void cvrf_element_to_dom(const char *elm_name, const char *elm_value, xmlNode *p
 	xmlNode *elm_node = xmlNewTextChild(parent, NULL, BAD_CAST elm_name, BAD_CAST elm_value);
 }
 
-/*
-void cvrf_index_export_xml(struct cvrf_index *index, struct oscap_source *export_source) {
-
-	__attribute__nonnull__(index);
-	__attribute__nonnull__(export_source);
+xmlNode *cvrf_index_to_dom(struct cvrf_index *index, xmlDocPtr doc, xmlNode *parent, void *user_args) {
```

Comment:
Please use xmlNodePtr, I have no clue why but this is what we do everywhere else.